addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -956,7 +1060,7 @@ QUnit.test('composer: send button is disabled if attachment upload is not finish
     );
 });
 
-QUnit.test('warning when attempting to post message when attachments are still uploading', async function (assert) {
+QUnit.test('warning when attempting to post message when attachments are still uploading with send shortcuts', async function (assert) {
```

Comment:
swap "when attachments are still uploading" and "with send shortcusts" (the latter referring to "post message", it's better next to it)